zerocopy-derive/tests/ui-nightly/enum.rs

```diff
@@ -114,6 +122,21 @@ enum FromZeros5 {
     B,
 }
 
+struct NotFromZeros;
+
+#[derive(FromZeros)]
+#[repr(u8)]
+enum FromZeros6 {
+    A(NotFromZeros),
+}
+
+#[derive(FromZeros)]
+#[repr(u8)]
+enum FromZeros7 {
+    A = 1,
+    B(NotFromZeros),
```

Comment:
Make this a type that does implement `FromZeros` to make it clear that we're testing the fact that there's no 0 discriminant?

Reply:
Done